bazel/variables.bzl

```diff
@@ -1,4 +1,34 @@
-DEFAULT_COPTS = [
+DEFAULT_WIN_COPTS = [
```

Comment:
why is this in the build file as well as the bzl file?

Reply:
DEFAULT_COPTS r build options for maldoca. while DEFAULT_MINI_CHROMIUM_WIN_COPTS r build options for windows.

i renamed DEFAULT_COPTS to DEFAULT_MALDOCA_COPTS to make it explicit.

Reply:
i would prefer keeping them separate (at least for now). however, followed your advice in option 2 and added comments to make it explicit.